deploy/helm/scf/assets/operations/instance_groups/bits.yaml

```diff
@@ -31,7 +31,7 @@
     namespace: {{ .Release.Namespace }}
 - type: replace
   path: /variables/name=bits_service_ssl/options/signer_type?
-  value: cluster
+  value: local
```

Comment:
Can you please explain this?
From our tests, using a Kube CSR was the only way to get the nodes to trust our registry.

Reply:
* In order to make push work from the outside the bit-service must be reachable, which is achieved via the `gorouter`. But the `gorouter` does not trust the `cluster` CA.
* To keep it possible for the `kubelet` to pull the image, we changed it to pull the image using the external url (i.e. it also goes thru the `gorouter`).

Reply:
I don't think your first statement is correct. The bits service should _only_ be used as a registry for Eirini docker images.
Pulling the image through the gorouter means the Kube nodes need to trust its CA - how does that happen?